gcn_monitor/cli.py

```diff
@@ -40,7 +40,8 @@ def host_port(host_port_str):
     show_default=True,
     help="Log level",
 )
-def main(prometheus, loglevel):
+@click.option("--bucketarn", help="Bucket ARN")
```

Comment:
```suggestion
@click.option("--bucket-arn", help="Bucket ARN")
```

And probably the argument name changes to `bucket_arn`.